llvm/test/Transforms/LoopVectorize/AArch64/llvm.sincos.ll

```diff
@@ -0,0 +1,170 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "(:|sincos|extractvalue|store)" --version 5
```

Comment:
nit: Can you remove this NOTE before landing because it looks like you've actually edited the autogen output.

Reply:
The `CHECK:` lines are automatically generated (without any manual edits) and still can be updated by the script without altering the `CHECK-COST:` lines (which are manual). 

This "NOTE:" line also includes the update script args `UTC_ARGS: --filter "(:|sincos|extractvalue|store)"` which means any future updates will be configured the same way.

Reply:
(the `update_test_checks.py` script automatically applies the arguments from the `UTC_ARGS:` section)

Reply:
Ah my bad. Apologies for the spam! I hadn't spotted the filter args. :face_palm